cms/djangoapps/contentstore/views/tabs.py

```diff
@@ -141,8 +141,11 @@ def reorder_tabs_handler(course_item, tabs_data, user):
 
     # the old_tab_list may contain additional tabs that were not rendered in the UI because of
     # global or course settings.  so add those to the end of the list.
+    # if there any non static tab in the old_tab_list, they should not be added to the new_tab_list
     non_displayed_tabs = set(old_tab_list) - set(new_tab_list)
-    new_tab_list.extend(non_displayed_tabs)
+    for tab in non_displayed_tabs:
+        if isinstance(tab, StaticTab):
+            new_tab_list.append(tab)
```

Comment:
Unfortunately, what this code will do is remove these tabs permanently from the course! This will break the course. 

Currently if you reorder tabs, you will end up with a broken course.